packages/rlc-codegen/src/buildPaginateHelper.ts

```diff
@@ -11,6 +11,7 @@ export function buildPaginateHelper(model: RLCModel) {
   }
   let file: string = readFileSync(
     path.join(__dirname, "static", "paginateHelper.ts.hbs"),
```

Comment:
@joheredi Could you take a look at the issue caused by `__dirname`? 

[build failure](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1849219&view=logs&j=bc242a59-cf42-53cf-4020-48ac851477f9&t=0001b62a-57b2-5689-389c-0ce61249e0ad&l=22)

**How to reproduce:**

1. rush update && rush build
2. cd packages/cadl-rlc-test
3. rushx smoke-test